sdk/test-utils/test-utils/test/tracing/azureTracing.ts

```diff
@@ -0,0 +1,129 @@
+import { OperationTracingOptions, useInstrumenter } from "@azure/core-tracing";
+import { assert } from "chai";
+import { TestInstrumenter } from "../../src";
+import { SpanGraph, SpanGraphNode } from "../../src/tracing/spanGraphModel";
+// this is the plugin used in the test file
+function chaiAzureTrace(chai: Chai.ChaiStatic, _utils: Chai.ChaiUtils) {
```

Comment:
I wonder if we should just call it `chaiAzure` instead of `chaiAzureTrace` because we can plug-in any number of custom assertions...